examples/web/index.html

```diff
@@ -501,6 +501,175 @@ <h2 class="typography-heading-medium-text">Alert</h2>
+              alt="Spirit"
+            />
+            <div class="Header__toggle">
+              <button
+                type="button"
+                class="Button Button--inverted"
+                data-toggle="header"
+                data-target="#header-example-1__bar"
+                aria-expanded="false"
+              >
+                <span class="Header__icon Header__icon--menu" aria-hidden="true"></span>
+                <span class="accessibility-hidden">Menu</span>
+              </button>
+            </div>
+            <nav id="header-example-1__bar" class="Header__bar">
+              <div class="Header__barContent">
+                <button
+                  type="button"
+                  class="Header__close"
+                  data-toggle="header"
+                  data-target="#header-example-1__bar"
+                  aria-expanded="false"
+                >
+                  <span class="Header__icon Header__icon--close" aria-hidden="true"></span>
+                  <span class="accessibility-hidden">Close</span>
+                </button>
+                <div class="Header__actions Header__actions--primary">
+                  <ul class="Header__nav">
+                    <li class="Header__navItem Header__navItem--current">
+                      <a href="#" class="Header__navLink">Nabídky práce</a>
+                    </li>
+                    <li class="Header__navItem">
+                      <a href="#" class="Header__navLink">Brigády</a>
+                    </li>
+                    <li class="Header__navItem">
+                      <a href="#" class="Header__navLink">Inspirace</a>
+                    </li>
+                    <li class="Header__navItem">
+                      <a href="#" class="Header__navLink">Historie odpovědí</a>
+                    </li>
+                    <li class="Header__navItem">
+                      <a href="#" class="Header__navLink">Zaměstnavatelé</a>
+                    </li>
+                  </ul>
+                </div>
+                <div class="Header__actions Header__actions--secondary">
+                  <a href="#" class="Button Button--primary">Přihlásit</a>
+                  <a href="#" class="Button Button--inverted">Vstup pro firmy</a>
+                </div>
+              </div>
+            </nav>
+          </header>
+
+          <script>
+            const HEADER_TOGGLE_SELECTOR = '[data-toggle="header"]';
+            const HEADER_BREAKPOINT = 1280;
+            const OPEN_CLASSNAME = 'is-open';
+
+            window.addEventListener('DOMContentLoaded', () => {
```

Comment:
We are still on ES5 right now, so I suggest using `function` instead of the arrow function. Or had we made the decision to drop support of IE?

Reply:
That is applied also for `const`.

Reply:
> Or had we made the decision to drop support of IE?

Yes, [we have](https://github.com/lmc-eu/code-quality-tools/blob/main/packages/browserslist-config/index.js).